runner/runners/invoke.go

```diff
@@ -171,9 +171,13 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 	// Wait for process to complete (or cancel if we're told to)
 	select {
 	case <-abortCh:
+		stdout.Write([]byte(fmt.Sprintf("%s\n\nTask aborted: %v", marker, cmd.String())))
+		stderr.Write([]byte(fmt.Sprintf("$s\n\nTask aborted: %v", marker, cmd.String())))
```

Comment:
$s

Also, if it isn't a pain, can you paste what the output looks like / does it catch the attention enough?